pyccel/codegen/printing/ccode.py

```diff
@@ -404,9 +404,29 @@ def _print_PythonAbs(self, expr):
             self._additional_imports.add("complex")
             func = "cabs"
         else:
-            func = "abs"
+            func = "labs"
         return "{}({})".format(func, self._print(expr.arg))
 
+    def _print_PythonMin(self, expr):
+        arg = expr.args[0]
+        if arg.dtype is NativeReal() and len(arg) == 2:
+            self._additional_imports.add("math")
+            return "fmin({}, {})".format(self._print(arg[0]),
+                                         self._print(arg[1]))
+        else:
+            return errors.report("min not yet supported in C", symbol=expr,
+                    severity='fatal')
```

Comment:
can you use a meaningful error message like: `min of {type} not yet supported in C`.

Reply:
I can see why you want this but I think ` min of {type} not yet supported in C` is just as misleading. Min is not fully supported for any types. There is support for the basic case of min with dtype = float and only 2 arguments but this is very specific so I'm not sure it's worth detailing in the error message

Reply:
Fixed in c6c74a3